.gitignore

```diff
@@ -33,3 +34,6 @@ yarn-error.log*
 .idea/
 .log/
 *.bak
+
+# ignore shared code packages build files
+packages/*/*.js
```

Comment:
Could we exclude the build folders instead if it's easy?

Reply:
yup. done. good idea